app/graphql/types/subscription_type.rb

```diff
@@ -14,4 +14,8 @@ def weather_published(latitude:, longitude:); end
   field :announcement_published, Types::AnnouncementType, description: "An announcement was published"
 
   def announcement_published; end
+
+  field :schema_sha, String, description: "Updated SHA value"
```

Comment:
`schema` is misleading since this is not the database version. Its the deployment sha or application sha.

Reply:
Yes, but doesn't this field statement produce a subscription type in the GraphQL database? I agree it is the application sha on the server side, so renaming it to something like `deployment_sha` makes sense.